code/modules/antagonists/blob/overmind.dm

```diff
@@ -64,7 +64,7 @@ GLOBAL_LIST_EMPTY(blob_nodes)
 	name = new_name
 	real_name = new_name
 	last_attack = world.time
-	var/datum/blobstrain/BS = pick(GLOB.valid_blobstrains)
+	var/datum/blobstrain/BS = /datum/blobstrain/reagent/distributed_neurons //BUBBERSTATION CHANGE, YOUR STRAIN WILL ALWAYS BE THE SAME.
```

Comment:
Not too sure if distributed neurons should be the pick, reactive spines feel like a generally safer starter.